src/cascadia/TerminalControl/SearchBoxControl.cpp

```diff
@@ -0,0 +1,239 @@
+﻿// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "SearchBoxControl.h"
+#include "SearchBoxControl.g.cpp"
+
+using namespace winrt;
+using namespace winrt::Windows::UI::Xaml;
+using namespace winrt::Windows::UI::Core;
+
+namespace winrt::Microsoft::Terminal::TerminalControl::implementation
+{
+    // Constructor
+    SearchBoxControl::SearchBoxControl() :
+        _goForward(false),
+        _isCaseSensitive(false)
+    {
+        InitializeComponent();
+
+        this->CharacterReceived({ this, &SearchBoxControl::_CharacterHandler });
+
+        _goForwardButton = this->FindName(L"SetGoForwardButton").try_as<Controls::Primitives::ToggleButton>();

```

Comment:
Is it not possible to use the names bound in by the Xaml compiler? `_GoForwardButton()` should exist already. Maybe I don't understand properly?